python/tests/test_account.py

```diff
@@ -938,6 +938,30 @@ def test_send_and_receive_message_markseen(self, acfactory, lp):
         except queue.Empty:
             pass  # mark_seen_messages() has generated events before it returns
 
+    def test_reply_privately(self, acfactory):
+        ac1, ac2 = acfactory.get_two_online_accounts()
+
+        group1 = ac1.create_group_chat("group")
+        group1.add_contact(ac2)
+        group1.send_text("hello")
+
+        msg2 = ac2._evtracker.wait_next_messages_changed()
+        group2 = msg2.create_chat()
+        assert group2.get_name() == group1.get_name()
+
+        msg_reply = Message.new_empty(ac2, "text")
+        msg_reply.set_text("message reply")
+        msg_reply.quote = msg2
+
+        private_chat = ac2.create_chat(ac1)
+        private_chat.send_msg(msg_reply)
+
+        private_chat1 = ac1.create_chat(ac2)
```

Comment:
It's probably better to create this chat before sending message, otherwise there is a race condition: sometimes you test reception into created chat, sometimes message arrives before chat is created.